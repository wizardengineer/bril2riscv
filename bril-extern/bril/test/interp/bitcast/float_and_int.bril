@main() {
  as_int: int = const 4591870180066957722;
  as_float: float = bits2float as_int;
  back_to_int: int = float2bits as_float;
  print as_int as_float back_to_int;
}
