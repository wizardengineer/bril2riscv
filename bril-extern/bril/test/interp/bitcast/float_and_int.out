4591870180066957722 0.10000000000000001 4591870180066957722
